### ISSUE TITLE
Screen reader focus does not shift to "Contact information" after clicking on the skip link—it stays on the skip link

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/accessliaison#33]
Switch focus to Contact information. A screen reader should announce "footer" when the focus is switched, and when "read from here" command is given it should read the contact info first. Good example: skip link on WCAG
